components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASLoginDataPublisherImpl.java

```diff
@@ -199,11 +211,21 @@ protected void publishAuthenticationData(AuthenticationData authenticationData)
 
     }
 
+    @Override
+    public void init(InitConfig initConfig) {
+
+    }
+
     @Override
     public String getName() {
         return AuthPublisherConstants.DAS_LOGIN_PUBLISHER_NAME;
     }
 
+    @Override
+    public int getPriority(MessageContext messageContext) {
+        return 0;
```

Comment:
Any reason to implement this method ? Why do we have to use a hard coded value 0 ?